examples/shader/shader_material.rs

```diff
@@ -73,8 +73,13 @@ impl RenderAsset for CustomMaterial {
         (render_device, material_pipeline): &mut SystemParamItem<Self::Param>,
     ) -> Result<Self::PreparedAsset, PrepareAssetError<Self::ExtractedAsset>> {
         let color = Vec4::from_slice(&extracted_asset.color.as_linear_rgba_f32());
+
+        let byte_buffer = [0; <Vec4 as encase::Size>::SIZE.get() as usize];
```

Comment:
```suggestion
        let byte_buffer = [0u8; <Vec4 as encase::Size>::SIZE.get() as usize];
```

Also, this seems to be a very common pattern that is possibly worth a small helper function?

Reply:
Indeed (for small writes).

Reply:
It seems it can't be made into a generic function because you can't (yet) use associated constants in const expressions (for the array length).

Reply:
Ok, no problem. It's not a big deal.